### ISSUE TITLE
.ddev added to git on commit in project. Ask to add .ddev to gitignore?

### ISSUE THREAD

**Issue by JPustkuchen** [webksde/ddev-vscode-devcontainer-drupal9-template#63]
**Problem:**
When using git in the project, .ddev is by default added to the git repository. Perhaps we should ask to add it to git ignore... or discuss how to work with git in ddev projects in general, perhaps the overall logic is wrong...

As a workaround for existing projects, I added .ddev to our default .gitignore file.

**Comment by joshsedl** [comment]
I think we should just always delete the .git and .gitignore files and never allow them to be there after using "drowl-init". Since there is no reason to keep the git files besides development. When keeping the git files after setup, and we push anything to this repository, the complete repo file structure will be invalid.

Alternatively, we could also create a complex .gitignore file to filter out all files created on "drowl-init". But I think this is not necessary, and we should rather just ALWAYS delete the git files and don't even ask anymore.

**Comment by JPustkuchen** [comment]
@joshsedl I agree to remove them on drowl-init... they are only helpful for development of this REPO, which stands against the drowl-init use :)
